tendermint/src/types/proto/validator.rs

```diff
@@ -1,4 +1,5 @@
 use address::ValAddress;
+use ibc_proto::protobuf::Protobuf;
```

Comment:
Does it differ to `core::Protobuf`?

Reply:
As in the one in Gears? I don't think so, but this crate doesn't depend on Gears.

Reply:
Oh, my bad. I didn't noticed that.